examples/app-with-on-behalf-of-auth/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '3.0'
+
+services:
+  grafana:
+    container_name: 'grafana-appwithonbehalfofauth-app'
+    build:
+      context: ./.config
+      args:
+        grafana_version: ${GRAFANA_VERSION:-9.5.3}
```

Comment:
nit: if this requires 10.1 should we not set that as the default value? @andresmgot

Reply:
we could also apply the change from https://github.com/grafana/grafana-plugin-examples/pull/175/files to make the image flexible

Reply:
:+1: I didn't pay attention to this docker-compose. I'll try to update it to make it work out of the box